spec/features/search/view_search_results_spec.rb

```diff
@@ -24,13 +24,8 @@
       expect(page).to have_content('2000 Pilot Environmental Sustainability Index (ESI)')
 
       # Temporal Extent
-      expect(page).to have_content('1978-01-01 to 1999-12-31')
-
-      # Spatial Extent
-      expect(page).to have_content('W: -180.0')
-      expect(page).to have_content('N: 90.0')
-      expect(page).to have_content('E: 180.0')
```

Comment:
I think you can still look for these spatial values, just look for -180.0 and the other lat/longs